instrumentation-api-semconv/src/test/java/io/opentelemetry/instrumentation/api/instrumenter/messaging/MessagingAttributesExtractorTest.java

```diff
@@ -174,5 +175,11 @@ public Long messagePayloadCompressedSize(Map<String, String> request) {
     public String messageId(Map<String, String> request, String response) {
       return response;
     }
+
+    @Nullable
+    @Override
+    public String messagePayload(Map<String, String> stringStringMap) {
+      return null;
+    }
```

Comment:
Would be cleaner if you set this as the default implementation in the interface itself instead of in every instrumentation. See my PR in `instrumentation-api-semconv/src/main/java/io/opentelemetry/instrumentation/api/instrumenter/http/HttpCommonAttributesGetter.java` for an example.

Reply:
i.e.
```
@Nullable
  default Map<String, String> requestHeaders(REQUEST request, @Nullable RESPONSE response) {
    return Collections.emptyMap();
  }
...

Reply:
Cool! wasn't aware of this java feature